sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -2772,6 +2773,39 @@ class DataSourceV2SQLSuiteV1Filter
     }
   }
 
+  test("SPARK-XXXXX: test column stats") {
```

Comment:
Could you use SPARK-41378 instead of SPARK-XXXXX?

Reply:
Fixed. Thanks